javalib/src/main/scala/java/nio/HeapByteBuffer.scala

```diff
@@ -8,122 +8,167 @@ private[nio] final class HeapByteBuffer private (
   position(_initialPosition)
   limit(_initialLimit)
 
+  private[this] implicit def newHeapByteBuffer = HeapByteBuffer.NewHeapByteBuffer
+
```

Comment:
For some reason known only of scalac, the implicit is not picked up if it's an import.